yoti/readme.txt

```diff
@@ -84,7 +84,7 @@ Here you can find the changes for each version:
 
 Release Date - January 2020
```

Comment:
Presume we'll change this to tomorrow if it goes out tomorrow?

Reply:
Yes, will update on #70